File: 252-meeting-rooms/252-meeting-rooms.java
class Solution {
    public boolean canAttendMeetings(int[][] intervals) {
        Arrays.sort(intervals, (a, b) -> a[1] - b[1]);
        int end = -1;
        for(int[] interval : intervals) {
            if(interval[0] < end)
                return false;
            end = interval[1];
        }
        return true;
    }
}

File: 252-meeting-rooms/252-meeting-rooms.py
class Solution:
    def canAttendMeetings(self, intervals: List[List[int]]) -> bool:
        intervals=sorted(intervals, key=lambda x: x[0])
        end = float('-inf')
        for i in range(len(intervals)):
            if intervals[i][0] < end: return False
            end = intervals[i][1]
        return True